in_toto/models/layout.py

```diff
@@ -38,7 +38,7 @@
 from dateutil.parser import parse
 
 import in_toto.matchrule_validators
-import securesystemslib.exceptions
+from securesystemslib.exceptions import FormatError
```

Comment:
`FormatError` is not a module, it is a class of the module `securesystemslib.exceptions`.
So you have import the module:
```python
import securesystemslib.exceptions
```
and then use the class specifying the full path, e.g.
```python
raise securesystemslib.exceptions.FormatError("This is a format error!")
```